### PR TITLE
First html page

### PR DESCRIPTION
**Pull request by JohnOldman2018**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,10 +29,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
-		<!--<dependency>
+	<!--	<dependency>
```
Comment (amussagaliyev) [inline pom.xml:32]: What are these changes? Please discard them

File: src/main/java/com/jantoleu/rooms_manager/controllers/RoomController.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class RoomController {
+
+    @GetMapping("/greeting")
```
Comment (amussagaliyev) [inline src/main/java/com/jantoleu/rooms_manager/controllers/RoomController.java]: let's make the path index.html

File: src/main/java/com/jantoleu/rooms_manager/controllers/RoomController.java [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    public String greeting(@RequestParam(name="name", required=false, defaultValue="World") String name, Model model) {
+        model.addAttribute("name", name);
+        return "greeting";
```
Comment (amussagaliyev) [inline src/main/java/com/jantoleu/rooms_manager/controllers/RoomController.java]: the string returned from this method should be the name of template.
But there is no template with name greetings.html
Please fix

File: src/main/resources/templates/main_page.html [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```
Comment (amussagaliyev) [inline src/main/resources/templates/main_page.html]: did't we agree to name this file index.html?

**Review by amussagaliyev — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (JohnOldman2018) [reply pom.xml:32]:
```suggestion
	<!--	<dependency>
```

File: src/main/java/com/jantoleu/rooms_manager/controllers/RoomController.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class RoomController {
+
+    @GetMapping("/index")
```
Comment (amussagaliyev) [inline src/main/java/com/jantoleu/rooms_manager/controllers/RoomController.java:11]: Let's have .html for endpoints serving pages

File: pom.xml (hunk 1 above)
Comment (amussagaliyev) [reply pom.xml:32]: pom.xml shouldn't be in changed files at all

File: src/main/resources/templates/index.html [hunk 6]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta content="text/html" charset="UTF-8" http-equiv="Content-Type">
+    <title>Room Lighting Manager</title>
+</head>
+<body>
+<h1>Hello the Great World of Programming!</h1>
+<p th:text="'Hello, ' + 'Jan' + '!'" />
```
Comment (amussagaliyev) [inline src/main/resources/templates/index.html:9]: Where did the `name` parameter go? Return it back please

**Review by amussagaliyev — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/index.html (hunk 6 above)
Comment (JohnOldman2018) [reply src/main/resources/templates/index.html:9]: ok

Comment (JohnOldman2018) [reply src/main/resources/templates/index.html:9]: fixed

File: src/main/java/com/jantoleu/rooms_manager/controllers/RoomController.java (hunk 5 above)
Comment (JohnOldman2018) [reply src/main/java/com/jantoleu/rooms_manager/controllers/RoomController.java:11]: fixed

**Pull request merged**